assets/src/constants.js

```diff
@@ -27,6 +27,9 @@ import Slabo27 from '../images/font-names/slabo-27.svg';
 import SourceSansPro from '../images/font-names/source-sans-pro.svg';
 import Ubuntu from '../images/font-names/ubuntu.svg';
 
+export const STORY_PAGE_INNER_WIDTH = 328;
```

Comment:
FWIW there are similar constants in the `EditorCarousel` component. We might be able to move them to this file and calculate them based on `STORY_PAGE_INNER_WIDTH`

These constants are used to calculate the `translateX` offset in the horizontal page navigation.

https://github.com/ampproject/amp-wp/blob/f5289bbd12be66ad8b53f39631e390004fd2d759/assets/src/components/editor-carousel/index.js#L16-L34

Reply:
Thank you for noting this. I noticed this, too, in case of positioning, the inner width is needed, it looks like in `EditorCarousel` it's together with the 5px border. I'll come back to this later, if you find anything else in the PR that needs attention then do let me know. 👍